### PR TITLE
Add check_version function to use in scripts for mewarpx

### PR DESCRIPTION
**Pull request by peterscherpelz**
I even included a unit test! And cleaned up util.py, which had a duplicated function presumably from a past unfortunate merge.

### CODE REVIEW

File: mewarpx/mewarpx/utils_store/util.py [hunk 1]
```diff
@@ -20,6 +22,54 @@
...
 
+def check_version(script_version, script_physics_version):
+    """Check that a run script is compatible with this mewarpx.
+
+    If this mewarpx distribution is *lower* than that of the run script, throw
+    an error. If this mewarpx distribution API or physics version is higher
+    than that of the run script, throw a warning. Else do nothing.
+
+    Arguments:
+        script_version (tuple): A tuple of ints representing the metools
```
Comment (roelof-groenewald) [inline mewarpx/mewarpx/utils_store/util.py]:
```suggestion
        script_version (tuple): A tuple of ints representing the mewarpx
```

File: mewarpx/mewarpx/utils_store/util.py [hunk 2]
```diff
@@ -20,6 +22,54 @@
...
+            version of the run script: (API_version, feature_version,
+            patch_version).
+    """
```
Comment (roelof-groenewald) [inline mewarpx/mewarpx/utils_store/util.py:37]:
```suggestion
        script_physics_version (int): Integer representing the physics version of the run script.
    """
```

**Review by roelof-groenewald — CHANGES_REQUESTED** [review]
Just two small things. 
Also I guess we should now start updating the changelog and version number with every PR?

**Review by roelof-groenewald — APPROVED** [review]

**Review by KZhu-ME — APPROVED** [review]
Looks good!

**Pull request merged**